### ISSUE TITLE
[Feature]: Chat archive / archiving

### ISSUE THREAD

**Issue by Mikaela** [simplex-chat/simplex-chat#2541]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Platform

Android, iOS

### App version

5.1.2

### Feature

I would like to have an archive when I scroll the chat list up and be able to throw chats there at will out of cluttering the main view which currently has all chats.

If there was new activity in the chats and they were unmuted, they should return to the main list.

My reasoning is having multiple chats with the same users or pending #444 groupchats which have just my devices and the other persons devices, so I don't need to see all contacts in the same view. I know I could just delete them, but then I would lose history that is not yet expired and then I might need to recreate the contacts/chats in the future.

**Comment by EchedeyLR** [comment]
I would also be interested in this feature.

It also helps to avoid addiction at first view.

**Comment by epoberezkin** [comment]
> It also helps to avoid addiction at first view.

can you please explain?

**Comment by Mikaela** [comment]
>> It also helps to avoid addiction at first view.
> 
> can you please explain?

I refer to mindless doomscrolling or seeing that there is new activity and then just starting to read it instead of doing something more productive or healthy (or both).

**Comment by teqexplorer** [comment]
I agree; the chats list is getting far too long, a bit of information overload there, so it would be great if we can have a function/"List" called "Archived", which could function like a list, but with the extra effect of NOT showing those chats in the main list.

The main point is to get some of the chats out of sight, for efficiency.
